runtime_util/runtime_util.c

```diff
@@ -272,6 +272,27 @@ int8_t get_param_idx(uint8_t dev_type, char* param_name) {
     return -1;
 }
 
+param_id_t* get_params_to_kill(uint8_t* num_devices_with_params_to_kill) {
+    // Change this array as necessary
+    /**
+     * Parameters that should set to (and remain) 0, 0.0, or False, depending on the type,
+     * when the Robot should be emergency stopped.
+     * Ex: Robot may be emergency stopped if no user input is connected during TELEOP.
+     */
+    param_id_t params_to_kill[] = {
```

Comment:
Make static, or make it a global variable probably.

Reply:
Is it possible to make it static if it uses functions? It used to be a global variable and I moved it into a function because I was getting errors/warnings

Reply:
I'm getting `initializer element is not constant` for using things like `KoalaBear.type` and `get_param_idx()` if I try to make the array static